src/executors/default.yml

```diff
@@ -1,3 +1,4 @@
 description: A debian-based machine executor
 
-machine: true
+machine: 
```

Comment:
The lint job is failing on this line. It seems like there is a trailing space: https://app.circleci.com/pipelines/github/CircleCI-Public/gcp-gcr-orb/219/workflows/dc66c34c-b148-4155-9abe-6e3efe669484/jobs/710?invite=true#step-104-5.